jboss-eap-clustered-vmss-rhel8/README.md

```diff
@@ -25,15 +25,14 @@
 
 ## Solution Overview
 
-JBoss EAP (Enterprise Application Platform) is an open source platform for highly transactional, web-scale Java applications. EAP combines the familiar and popular Jakarta EE specifications with the latest technologies, like Microprofile, to modernize your applications from traditional Java EE into the new world of DevOps, cloud, containers, and microservices. EAP includes everything needed to build, run, deploy, and manage enterprise Java applications in a variety of environments, including on-premise, virtual environments, and in private, public, and hybrid clouds.
+JBoss EAP (Enterprise Application Platform) is an open source platform for highly transactional, web-scale Java applications. EAP combines the familiar and popular Jakarta EE specifications with the latest technologies, like Microprofile, to modernize your applications from traditional Java EE into the new world of DevOps, cloud, containers, and microservices. EAP includes everything needed to build, run, deploy, and manage enterprise Java applications in a variety of environments, including on-premises, virtual environments, and in private, public, and hybrid clouds.
 
-Red Hat Subscription Management (RHSM) is a customer-driven, end-to-end solution that provides tools for subscription status and management and integrates with Red Hat's system management tools. To obtain an rhsm account for JBoss EAP, go to: www.redhat.com.
+Red Hat Subscription Management (RHSM) is a customer-driven, end-to-end solution that provides tools for subscription status and management and integrates with Red Hat's system management tools. To obtain an RHSM account for JBoss EAP, go to: www.redhat.com.
 
-This Azure quickstart template creates all of the compute resources to run JBoss EAP 7.2 cluster running on RHEL 8.0 VMSS instances where user can decide the number of instances to be deployed and scale it according to their requirement.
 
 ## Template Solution Architecture
 
-This template creates all the Azure compute resources to run JBoss EAP 7.2 cluster on top RHEL 8.0 VMSS instances where user can decide the number of instances to be deployed and scale it according to their requirement. The following resources are created by this template:
+This Azure Resource Manager (ARM) template creates all the Azure compute resources to run JBoss EAP 7.2 cluster on top of RHEL 8.0 VMSS instances where the user can decide the number of instances to be deployed and scale it according to their requirement. The following resources are created by this template:
```

Comment:
We have changed this line in the new Draft in https://github.com/SpektraSystems/redhat-mw-cloud-quickstart/blob/master/jboss-eap-standalone-rhel7/README.md. Please review and let us know the feedback, so that we can replicate the same format here